src/components/FormSections/PersonalInfo.jsx

```diff
@@ -321,16 +370,20 @@ function PersonalInfo(props){
                         aria-hidden="true"
                         value={props.fieldData.email_address}
                         onChange={props.saveFieldData('email_address')}
-                        onBlur={(e) => setHandleErrors({ ...handleErrors, email_address: checkForErrors(e, 'check value length') })}
+                        onBlur={(e) => toggleError(e, checkForErrors(e, 'check value length'))}
+                        onInvalid={(e) => e.target.setCustomValidity(' ')}
+                        onInput={(e) => e.target.setCustomValidity('')}
                     />
                 </div>
             </Grid>
 
             {raceFieldState && (
                 <Grid row gap>
                     <Grid tablet={{ col: 4 }}>
-                        <div className={((parseInt(raceFieldState.required)) && handleErrors.race) ? 'error-container' : ''}>
-                            <Label className="text-bold" htmlFor="race-ethic-group-select">{raceField.label}{(raceFieldState.required === "1") && <span className='required-text'>*</span>}
+                        <div className="input-parent">
+                            <Label className="text-bold" htmlFor="race-ethic-group-select">
```

Comment:
race ethnic field displaying error onblur even if it's not required.
![Screenshot 2024-02-05 at 4 19 19 PM](https://github.com/usagov/vote-gov-nvrf-app/assets/19731897/b84a95cc-a9e9-4918-ae17-79fcee6bd4a8)